### PR TITLE
1354 map link

### PR DESCRIPTION
**Pull request by mtlaney**
﻿## Summary
Addresses Issue #1354 

Please note if fully resolves the issue per the acceptance criteria: Yes

*Updating ESRI maps to include links to forests with fuelwood permits. Coding style is dated in order to function correctly with esri popuptemplate. The esri popuptemplate function call does not accept additional parameters or fat arrow functions*

## Impacted Areas of the Site
- /home

## This pull request is ready to merge when...
- [ ] Feature branch is starts with the issue number
- [ ] Is connected to its original issue via zenhub 👇
- [ ] Tests have been updated 
- [ ] All tests are passing and meet coverage, linting, and accessibility requirements. And no security vulnerabilities ⚫️(Jenkins)
- [ ] This code has been reviewed by someone other than the original author

### CODE REVIEW

File: frontend/src/app/esri-map/esri-map.component.scss [hunk 1]
```diff
@@ -1,12 +1,10 @@
 /* import the required JSAPI css */
-// @import 'https://js.arcgis.com/4.10/esri/css/main.css';
-@import 'https://js.arcgis.com/4.10/esri/themes/dark-green/main.css';
+@import 'https://js.arcgis.com/4.10/esri/css/main.css';
 // @import '../../../node_modules/arcgis-js-api/themes/dark/main.scss';
```
Comment (Dmac26) [inline frontend/src/app/esri-map/esri-map.component.scss:3]: Do we want to leave in this commented out import statement?

File: frontend/src/app/esri-map/esri-map.component.ts [hunk 2]
```diff
@@ -12,17 +12,16 @@ export class EsriMapComponent implements AfterViewInit {
 
   @ViewChild('mapViewNode') private mapViewEl: ElementRef;
 
-  constructor(public router: Router) { }
+  constructor(
```
Comment (Dmac26) [inline frontend/src/app/esri-map/esri-map.component.ts:15]: Purely for my own education, why did you change the format here?

File: frontend/src/app/esri-map/esri-map.component.scss (hunk 1 above)
Comment (mtlaney) [reply frontend/src/app/esri-map/esri-map.component.scss:3]: Good catch, I left this in incase we want to revert back to the dark there post design review (It can be removed though)

File: frontend/src/app/esri-map/esri-map.component.ts (hunk 2 above)
Comment (mtlaney) [reply frontend/src/app/esri-map/esri-map.component.ts:15]: I typically break the constructor lines if we are adding multiple items. In this case I was importing firewoodInfoService to .getAll() and loop thru the forests that way, but esri hated that so I had to remove it
